### PR TITLE
Social media

### PR DESCRIPTION
**Pull request by MahmoudBakr23**
We wrote RSpec tests for:
1- User model
2- Post model
3- Comment model

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -51,6 +51,8 @@ Lint/RaiseException:
   Enabled: false
 Lint/StructNewOverride:
   Enabled: false
+Lint/Void:
+  Enabled: false
```
Comment (Dandush03) [inline .rubocop.yml:57]: ### You're not supposed to change lint configuration

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle: Request Changes :red_circle: 

### Hey guys, you've done a fantastic job :muscle: 
__Be sure to fix all the issue commented below before submiting a new review__

- [ ] You have a different version between your __`.ruby-version`__ file and the __`Gemfile`__. This can cause some internal errors while using __`bundler`__ with a different control environment like __`rbenv`__
- [ ] __You did a great job with your models but, in this milestone, you most implement does change into your views, be sure to add the followings:__

   

-  - [ ] User can invite friends 
-  - [ ] User can see friendships requests 
-  - [ ] User has to accept a friendship request to start a friendship
-  - [ ] User can reject a friendship request
-  - [ ] There is a button to invite the user to friendship on the user's list page and on the single-user page
- [ ] __Each module, view, or controller should be tested as require before__

# Happy Coding :muscle: :computer:

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello Team
### You have done well so far :clap: You are very close to getting it right:-
- [ ] In the profiles, add a section to list friends
- [ ] Install all the required dependencies and run stylelint command, [follow this guide](https://github.com/microverseinc/linters-config/tree/master/ror)
### Make the changes and submit for another code review. Good luck.
- Happy coding!

**Comment by MahmoudBakr23** [comment]
About the linters, everything is already in our repo!
.github/workflows/linters.yml
.rubocop.yml
.stylelintrc.json

**Review by rOluochKe — APPROVED** [review]
Hello Team
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**